### DISCUSSION TITLE
How can I help to translate ?

### DISCUSSION THREAD

**Discussion by berserkwarwolf** [ManifoldScholar/manifold#3494] [Q&A]
I can contribute with translation to Spanish and Portuguese. How can I help ? Where should I start ?

**Comment by fishfree** [comment]
I have the same question. Suggest to use Transifex or Weblate services.

**Comment by mkgold** [comment]
Thank you for your interest @berserkwarwolf @fishfree. We're doing some localization feature development for Manifold that will begin to allow us to take up offers like yours in the future. For now, our development team is still at work. Once that work is completed, we'll articulate a process (likely pull-request based) for accepting translations of the locale files that we create for English. We'll describe that process in our documentation once we are ready. For now, thank you for your interest in Manifold and please stay tuned!

**Comment by zdavis** [comment]
In v7, which will be released soon, just about all text strings will have been extracted from the client application. Once this version is out the door, we'll be looking for volunteers to copy the set of english strings (https://github.com/ManifoldScholar/manifold/tree/master/client/src/config/app/locale/en) and translate them into other languages. Our hope is that the community will provide these translations via a pull request against this repository. We'll be writing more about this as we approach the v7 release. 

There's a bit more work to be done on the API, which also returns some strings that need to be localized (error messages, for examples). That probably won't be done until v8 or v9, but it's on our radar.
